ArduPlane/mode_qstabilize.cpp

```diff
@@ -16,7 +16,7 @@ void ModeQStabilize::update()
 {
     // set nav_roll and nav_pitch using sticks
     // Scale from normalized input [-1,1] to target angles in centidegrees
-    const float roll_input = plane.channel_roll->norm_input();
+    const float roll_input = plane.channel_roll->get_control_in() / 4500.0f;
```

Comment:
please change pitch to be consistent, and add a note on why we're using get_control_in() (ie. because of the tailsitter swapping of axes)